src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/events/ActionPlanAppointmentEventPublisher.kt

```diff
@@ -70,7 +70,7 @@ class ActionPlanAppointmentEventPublisher(
   }
 
   private fun getAppointmentURL(session: DeliverySession): String {
-    val path = locationMapper.getPathFromControllerMethod(DeliverySessionController::getSessionForReferralId)
-    return locationMapper.expandPathToCurrentContextPathUrl(path, session.referral.id, session.sessionNumber).toString()
+    val path = locationMapper.getPathFromControllerMethod(DeliverySessionController::getDeliverySessionAppointment)
```

Comment:
@anthony-britton-moj At the moment, there are some appointment end point which are deprecated but we are still using it. Because changing involves changing lots of code in UI and we have it in our backlog. For the time being, we can retract back to how it was original. we can change it when we refactor the code